var/spack/repos/builtin/packages/meme/package.py

```diff
@@ -35,9 +35,20 @@ class Meme(AutotoolsPackage):
 
     version('4.11.4', '371f513f82fa0888205748e333003897')
 
+    variant('mpi', default=False, description='Enable MPI support')
```

Comment:
@adamjstewart shared memory parallelism isn't the only way to use this package but yes I can see how this should be the default given the package will enable mpi by default if it's found